components/chef-run/lib/chef-run/cli.rb

```diff
@@ -199,7 +200,11 @@ def perform_run
                                           config.delete(:protocol),
                                           config).targets
         temp_cookbook, initial_status_msg = generate_temp_cookbook(cli_arguments)
-        local_policy_path = create_local_policy(temp_cookbook)
+        local_policy_path = nil
+        UI::Terminal.render_job(TS.generate_policyfile.generating) do |reporter|
```

Comment:
Generating the Policyfile can sometimes take a few seconds, and right now it looks like the system hangs. So I also wanted to add a spinner around this process